scripts/deploy_cf.sh

```diff
@@ -64,4 +64,4 @@ cf_post_deploy() {
 
 cf_compile_manifest
 cf_deploy
-cf_post_deploy
+#cf_post_deploy
```

Comment:
Instead of commenting this out, can we delete entirely (including the function above).